src/averageAges.js

```diff
@@ -1,61 +1,63 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+const getSex = (people, sex) => people.filter(person => person.sex === sex);
+
+/* Calculate Men Average Age */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let isMen = getSex(people, 'm');
+
+  const isMenInCentury = isMen.filter(person =>
```

Comment:
```suggestion
  const menInCentury = ...
```

Reply:
you could follow this guideline:
boolean - for example, `isLoading`, `shouldUpdate`
single item – for example, `apple`, `box`, `student`
array of items – for example, `apples`, `boxes`, `studentsList`